### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by AkshithaReddy1899**
This PR is raised to rewrite the awesome book's code in modules.

# Things to do

- Use ES6 modules to write modular JavaScript.
- Learn how to use proper ES6 syntax.
- Understand improvements that ES6 brings to JavaScript.
- Use npm as a software packages system.

### CODE REVIEW

File: module/render.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    localStorage.setItem('booksList', JSON.stringify(this.arr));
+  }
+
+  pushListItem(item) {
+    const li = document.createElement('li');
+    li.className = 'list_item';
+
+    const p = document.createElement('p');
+    p.className = 'book-name';
+    p.innerHTML = `${item.bookName} by ${item.author}`;
```
Comment (brytebee) [inline module/render.js]: - [x] **Required:** The page should display, the Title and Author of every entry into the array, in your case collection. This is not the case at the moment. The Title is `undefined`. Let's fix this by ensuring the name(key) which is used to store data in the local storage is strictly equal to the one we use to get it. I hope this is helpful. 👍 👍

File: module/render.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+  }
+
+  // class to update array in the local storage
+
+  UpdateLocalStorage() {
+    localStorage.setItem('booksList', JSON.stringify(this.arr));
+  }
+
+  pushListItem(item) {
```
Comment (brytebee) [inline module/render.js:30]: - [ ] **Required:** To improve the user experience of your app, publish the exiting books every time the [DOM is loaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event). Also, [see](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) this resource for options and better understanding.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// event listener to trigger add class
+
+document.querySelector('.addBtn').addEventListener('click', () => {
+  collection.getBooks();
+  collection.addBooks();
+});
+
+// luxon time
+
+const time = DateTime.now();
```
Comment (brytebee) [inline index.js]: - [ ] **Required:** Let's try to implement the Luxon library as shown in the sneak peek. This is designed to help you practice how to work with libraries/frameworks by the following documentation. Take some time to study and play around with [this section of the guide](https://moment.github.io/luxon/#/formatting?id=intldatetimeformat).

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+- Luxon
+
+## License
+
+MIT
+
+---
+
+> GitHub [@AkshithaReddy1899](https://github.com) &nbsp;&middot;&nbsp;
+> Twitter [@Akshitha Reddy](https://twitter.com)
```
Comment (brytebee) [inline README.md]: - [ ] Properly document your work following the [Microverse template](https://github.com/microverseinc/readme-template). This helps you practise documentation as well as your communication (written) skills.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @AkshithaReddy1899👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- ✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @AkshithaReddy1899, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**